Tests/AccessibilityTests/accessibility-fss-ui-simplifiedsearch-page.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { chromium, BrowserContext, Browser, Page } from 'playwright'
+import { injectAxe,checkA11y } from 'axe-playwright'
+const { autoTestConfig } = require('../FunctionalTests/appSetting.json');
+const { pageObjectsConfig, pageTimeOut } = require('../FunctionalTests/pageObjects.json');
+import { LoginPortal} from '../FunctionalTests/helpermethod'
+
+let browser: Browser
+let context: BrowserContext;
```

Comment:
This context looks unused

Reply:
Removed unused context from the script. Thanks.